app/views/shared/_claim.html.haml

```diff
@@ -1,9 +1,24 @@
 %section
+
+  .certification-info-row
+    .bold-xsmall{ style: 'margin-bottom: 5px;' }
+      = t('.certification_box_title')
+    %div.bold-xsmall
```

Comment:
This line could just be ".bold-xsmall". HAML by default would create a div for any element that isn't explicitly mentioned. eg line 3. it would create a div with a class of "certification-info-row"